### DISCUSSION TITLE
simultaneous playback of only one youtube video

### DISCUSSION THREAD

**Discussion by tadmi** [vuepress-theme-hope/vuepress-theme-hope#2996] [Q&A] [answered]
Is it possible to stop the youtube video being played when another one is turned on?

It's about a situation where there are several youtube videos on the page.

How can I get to the [youtube player API?](https://developers.google.com/youtube/iframe_api_reference#Playback_status)

**Comment by Mister-Hope** [comment, marked as answer]
It's possible only if we use the api, currently we are using iframe to provide a tiny component, but yes we may support the feature you want if we import the iframe api. i believe it's possible for you to do that yourself by writing extra code in your markdown file and provide id or class for videos in page to reference them with css selector.

Further discussion is welcomed, but I need the gzip/origin size of the extra api chunk(performance consideration), and see if we can import it later in onMounted liftcycle without affecting current video play.

**Reply by Mister-Hope** [reply]
Maybe the behaviour for a single autoplay video or no autoplay video in page is easy. But we may need to discuss, what shall we do if there are multiple autoplay video detected in page? Also is the cost reasonable for us to provide a global pub/sub emitter together with a iframe api chunk only for play/pause consideration?

It's definitely easy for you to add some extra features based on your own need, but a developer needs more to think with when facing all users.
